braindecode/datautil/preprocess.py

```diff
@@ -251,3 +252,54 @@ def scale(data, factor):
     if hasattr(data, '_data'):
         data._data = scaled
     return scaled
+
+
+def filterbank(raw, frequency_bands, drop_original_signals=True, n_jobs=1,
+               method='fir', iir_params=None, phase='zero', fir_window='hamming',
+               fir_design='firwin', **filter_kwargs):
+    """Applies multiple bandpass filters to the signals in raw. Raw will be
+    modified in-place and number of channels in raw will be updated to
+    len(frequency_bands) * len(raw.ch_names) (-len(raw.ch_names) if
+    drop_original_signals). For filtering arguments please refer to
+    mne.filter.filter_data().
+
+    Params
+    ------
+    raw: mne.Raw
+        The raw singnals to be filtered
+    frequency_bands: list(tuple)
+        The frequency bands to be fitlered for (e.g. [(4, 8), (8, 13)])
+    drop_original_signals: bool
+        Whether to drop the original unfiltered signals
```

Comment:
missing many params

Reply:
In the docstring above I stated to refer to ```mne.filter.filter_data()``` for a description of the parameters. Should I copy the docstring from mne instead and paste it here?